Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.12.2-alpine3.19
+LABEL maintainer="sanyok.it@gmail.com"
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
The `COPY . .` command can lead to large image sizes if not managed carefully. Consider copying only necessary files to reduce the image size and improve build efficiency.